### ISSUE TITLE
Menu button no longer appears in Android Q

### ISSUE THREAD

**Issue by ywwg** [travistabbal/GrazeRSS#37]
Related to #35, but now this is a concrete problem.  It appears that Android is removing backwards-compatibility support for the old menu button -- it no longer appears in Android Q. So I can no longer access those menu options.

**Comment by nayfield** [comment]
does this happen if you have "System Navigation" set to "Two Button"?  (see [here](https://cdn.arstechnica.net/wp-content/uploads/2019/05/23-980x916.jpg)

**Comment by dasty** [comment]
Yes it happens to me too. Both in "Two Button", but also in "Three Button" too :(

**Comment by nayfield** [comment]
Sigh - that's the only way that "open in browser" or "share link" actually work since android 9.  (As other issues mention, long-press on a list item presents the option, but it crashes graze.).

@dasty - are you able to long-press on a list item, get a pop-up menu (maybe mis-sized?) and select "Mark Read Until Here" ... and get the expected result?  

I've had the Android 10 upgrade in my notification tray for a week but haven't done it.  It sounds like (at a minimum) I won't be able to:

- open in browser
- share link
- switch day/night mode

My first thought is that I don't do those things *that* often ... and that my most common use case of read feed text / maybe switch to graze web view / mark read until here would still survive...

But I also won't be able to open settings ever again ... and already can't install from the play store.  

I guess I'm going to have to give up.  But first (as a non java guy) will take a peek at the code.  Like the non-mechanic verifying that there's still an engine in his broken down car before panicking more.

**Comment by dasty** [comment]
>  are you able to long-press on a list item, get a pop-up menu (maybe mis-sized?) and select "Mark Read Until Here" ... and get the expected result?

@nayfield - tried it once and seem to work.

**Comment by glasser** [comment]
Seeing this too. No "open in browser" is tough :(

**Comment by BenjaminHCCarr** [comment]
Also seeing this....

The long press menu to share items is *VERY LIMITED*

**Comment by nayfield** [comment]
have some folks working on moving this to API 29 so we can get these fixed.  GrazeEleven is the name of my github project if you want to bookmark it .  on my first freelancer right now let's see how he does over the next week.
